### PR TITLE
Add docker scripts and just shim

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Description

This PR adds two docker scripts, `docker:build` and `docker:run`, to the pnpm `package.json` config. I've also replaced the uses of those docker commands in the README with the associated `pnpm` command.

I've also added a `justfile` to this repo so `just` can be used in lieu of `pnpm`. Since we use `just` for our other projects, this can help make the workflow similar to our other projects!

## Testing Instructions

Run the following:

1. `just`
1. `just run docker:build`
1. `just run docker:run`

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,10 @@ All files and folders should be written in `kebab-case`, with the exception of V
 | ------------ | ----------- | ----------- | ---------------------- |
 | /photos/\_id | /image/\_id | 301         | Nuxt server middleware |
 
+## Just
+
+If [`just`](https://github.com/casey/just) is your preferred command runner, you can also use `just run {script}` to run any pnpm script and `just` on its own to list the available scripts (e.g. `just run docker:build`).
+
```
Comment (zackkrida) [inline README.md:188]: Some mention here of the motivation might be nice, i.e. parity with the other Openverse repositories that use Just.

**Review by zackkrida — APPROVED** [review]
Love it.

**Review by krysal — APPROVED** [review]
Nice!

**Review by dhruvkb — APPROVED** [review]
The Docker-related scripts are a great addition. I'm not so sure about `just` but if others don't mind I don't either.

**Comment by AetherUnbound** [comment]
> I'm not so sure about `just` but I appreciate the cross-repo consistency and I don't mind it because `pnpm run` continues to work.

I find my muscle memory is so strong that every time I come to work on something on the frontend I'm always so surprised I can't use any of my `j` shortcuts 😅 The consistency is great for me since I use pnpm so infrequently!

**Pull request merged**